### PR TITLE
Refactor WorkspaceServiceTest to use the WorkspaceService API instead of the HTTP controller endpoints

### PR DESCRIPTION
**Pull request by wchamber**
The WorkspaceService test should test against the WorkspaceService's API, not the controller API. This is also much less verbose.

### CODE REVIEW

File: src/test/java/bio/terra/workspace/service/workspace/WorkspaceServiceTest.java [hunk 1]
```diff
@@ -71,125 +47,88 @@
...
-  public void testGetExistingWorkspace() throws Exception {
-    CreateWorkspaceRequestBody body = new CreateWorkspaceRequestBody();
+  public void testGetExistingWorkspace() {
     UUID workspaceId = UUID.randomUUID();
-    body.setId(workspaceId);
+    CreateWorkspaceRequestBody body = new CreateWorkspaceRequestBody().id(workspaceId);
 
-    CreatedWorkspace workspace = runCreateWorkspaceCall(body);
+    CreatedWorkspace workspace = workspaceService.createWorkspace(body, USER_REQUEST);
+    assertEquals(workspace.getId().toString(), workspaceId.toString());
```
Comment (zloery) [inline src/test/java/bio/terra/workspace/service/workspace/WorkspaceServiceTest.java:65]: Why call .toString() on both IDs?

**Review by zloery — APPROVED** [review]
LGTM

File: src/test/java/bio/terra/workspace/service/workspace/WorkspaceServiceTest.java (hunk 1 above)
Comment (wchamber) [reply src/test/java/bio/terra/workspace/service/workspace/WorkspaceServiceTest.java:65]: oops, done.

**Comment by davidangb** [comment]
@ungwudik I know this PR is already merged, but you had opinions on this code ... take a look?

**Comment by ungwudik** [comment]
> @ungwudik I know this PR is already merged, but you had opinions on this code ... take a look?

I looked at it yesterday (not necessarily as a reviewer, though). Thanks for bringing it up anyway.

**Pull request merged**